### PR TITLE
Add network put update

### PR DESCRIPTION
**Pull request by nvthongswansea**

### CODE REVIEW

**Review by itakouna — COMMENTED** [review]
would you consider NetworkUpdatePutRequest as a new feature that the customer should be aware of? If so, we need to add in to the changelog. all LGTM

**Comment by nvthongswansea** [comment]
@itakouna could you re-review and approve the PR please? I cannot merge the PR without an approval.

**Review by itakouna — APPROVED** [review]

**Pull request merged**
